src/crimson/os/seastore/cache.cc

```diff
@@ -996,7 +996,9 @@ record_t Cache::prepare_record(Transaction &t)
 	i->get_type(),
 	i->is_logical()
 	? i->cast<LogicalCachedExtent>()->get_laddr()
-	: L_ADDR_NULL,
+	: (is_lba_node(i->get_type())
+	  ? i->cast<lba_manager::btree::LBANode>()->get_node_meta().begin
+	  : L_ADDR_NULL),
```

Comment:
Looks to me `extent_t::addr` has commented an invariant that it is the "laddr of extent (L_ADDR_NULL for non-logical)". And cleaner does use this `extent_info_t::addr` to look up lba extents to check alive. The fix looks good to me, might need @athanatos to take another look.